app.rb

```diff
@@ -1,9 +1,22 @@
-require 'sinatra/base'
+require "sinatra/base"
+require "./lib/peep"
 
 class Chitter < Sinatra::Base
-  get '/test' do
-    'Test page'
+
+  get "/" do
```

Comment:
I would probably suggest using a different view for this reverse order of messages. You could try and call it peeps/all/reversed